prototypes/mms2dgls/mms2d_gls.output

```diff
@@ -0,0 +1,26 @@
+Running on 1 MPI rank(s)...
```

Comment:
Will need to be moved from prototypes to application_tests. You can call it mms_2d_simplex_gls or something.